.circleci/config.yml

```diff
@@ -94,6 +94,13 @@ jobs:
           target: test-sim-multi-seed-long
           description: "Test multi-seed simulation (long)"
 
+  test-cli:
+    executor: golang
+    steps:
+      - make:
+          target: test-sim
+          description: "Test client integration"
+
```

Comment:
```suggestion
```

We're migrating to github actions, this can  just be reverted

Reply:
Sure